src/shared/components/mutationTable/MutationTable.tsx

```diff
@@ -1381,7 +1449,11 @@ export default class MutationTable<
                 dataStore={this.props.dataStore}
                 downloadDataFetcher={this.props.downloadDataFetcher}
                 initialItemsPerPage={this.props.initialItemsPerPage}
-                initialSortColumn={this.props.initialSortColumn}
+                initialSortColumn={
+                    this.isAnnotationVisible
+                        ? this.props.initialSortColumn
+                        : MutationTableColumnType.CUSTOM_DRIVER
```

Comment:
In theory `props.initialSortColumn` can be any column, not just annotation column. By adding a condition like this we basically ignore the cases where `props.initialSortColumn` is not the annotation column.